lib/Slackbot_worker.js

```diff
@@ -358,6 +358,28 @@ module.exports = function(botkit, config) {
         bot.say(msg,cb);
     };
 
+    bot.startTyping = function(src) {
+        bot.reply(src, { type: 'typing' })
+    }
+
+    bot.replyWithTyping = function(src, resp, cb) {
+        var text;
+
+        if (typeof(resp) == 'string') {
+            text = resp;
+        } else {
+            text = resp.text;
+        }
+
+        var typingLength = 500 / 60 * text.length;
```

Comment:
You've assumed every letter takes 8.33 miliseconds to type. For a sentence of 10 characters it takes 83.33 ms which is faster then most of the humans. In order to make it more human noticable, I suggest you increase your typingLength value.